Dockerfiles/manifests/README.md

```diff
@@ -12,22 +14,26 @@ Create the appropriate ClusterRole, ServiceAccount, and ClusterRoleBinding:
 kubectl create -f stackstate-serviceaccount.yaml
 ```
 
-## Create manifest
-
-Replace <YOUR_API_KEY> with your Stackstate API key or use Kubernetes secrets to set your API key as an environment variable inside the `stackstate-agent.yaml`.
+### Enable Kubernetes state
 
-Deploy the DaemonSet with the command:
+To gather your kube-state metrics:
+* Download the [Kube-State manifests folder](https://github.com/kubernetes/kube-state-metrics/tree/master/kubernetes)
+* Apply them to your Kubernetes cluster:
 
 ```
-kubectl create -f stackstate-agent.yaml
+kubectl apply -f <NAME_OF_THE_KUBE_STATE_MANIFESTS_FOLDER>
 ```
 
-# Kubernetes Host Setup
+## Deploy the DaemonSet
 
-Installing the Agent directly on your host (rather than having the Agent run in a Pod) provides additional visibility into your ecosystem, independent of Kubernetes.
-To gather your kube-state metrics:
-* Download the [Kube-State manifests folder](https://github.com/kubernetes/kube-state-metrics/tree/master/kubernetes)
-* Apply them to your Kubernetes cluster:
-  ```
-  kubectl apply -f <NAME_OF_THE_KUBE_STATE_MANIFESTS_FOLDER>
-  ```
+Before deploying the agent there are few configuration settings to take care of, open the `stackstate-agent.yaml` and:
+
+* replace `<STACKSTATE_BACKEND_IP>` with your Stackstate backend IP
+* if you want to collect only containers information (and no processes) remove the env variable `DD_PROCESS_AGENT_ENABLED` (or set it to `false`)
```

Comment:
@TizianoPerrucci here if you set false, even the containers information we won't get it, we simply need to remove it to make it work.